modelbaker/processing/ili2db_algorithm.py

```diff
@@ -0,0 +1,324 @@
+            optional=True,
+        )
+        service_param.setHelp(self.tr("The PostgreSQL service config file."))
+        params.append(service_param)
+
+        host_param = QgsProcessingParameterString(
+            self.HOST,
+            self.tr("Host"),
+            defaultValue="localhost",
+            optional=True,
+        )
+        host_param.setHelp(
+            self.tr("The host of the database server. By default is localhost.")
+        )
+        params.append(host_param)
+
+        port_param = QgsProcessingParameterNumber(
+            self.PORT,
+            self.tr("Port"),
+            type=QgsProcessingParameterNumber.Type.Integer,
+            defaultValue=5432,
+            optional=True,
+        )
+        port_param.setHelp(
+            self.tr("The port of the database server. By default is 5432.")
+        )
+        params.append(port_param)
+
+        dbname_param = QgsProcessingParameterString(
+            self.DBNAME,
+            self.tr("Database"),
+            defaultValue=None,
+            optional=True,
+        )
+        dbname_param.setHelp(self.tr("The database name. The database should exist."))
+        params.append(dbname_param)
+
+        schema_param = QgsProcessingParameterString(
+            self.SCHEMA,
+            self.tr("Schema"),
+            defaultValue=None,
+            optional=True,
+        )
+        schema_param.setHelp(self.tr("The database schema."))
+        params.append(schema_param)
+
+        sslmode_param = QgsProcessingParameterEnum(
+            self.SSLMODE,
+            self.tr("SSL Mode"),
+            ["disable", "allow", "prefer", "require", "verify-ca", "verify-full"],
+            defaultValue=None,
+            optional=True,
+            usesStaticStrings=True,
+        )
+        sslmode_param.setHelp(self.tr("The SSL mode if needed."))
+        params.append(sslmode_param)
+        user_param = QgsProcessingParameterString(
+            self.USER,
+            self.tr("User"),
+            defaultValue=None,
+            optional=True,
+        )
+        user_param.setHelp(self.tr("The user to access the database."))
+        params.append(user_param)
+
+        password_param = QgsProcessingParameterString(
+            self.PASSWORD,
+            self.tr("Password"),
+            defaultValue=None,
+            optional=True,
+        )
+        password_param.setHelp(self.tr("The password of the user."))
+        params.append(password_param)
+
+        usesuperuser_param = QgsProcessingParameterBoolean(
+            self.USESUPERUSER,
+            self.tr("Use superuser login from the settings"),
+            defaultValue=False,
+        )
+        usesuperuser_param.setHelp(
+            self.tr("Excecute the task with the super user login from the settings")
+        )
+        params.append(usesuperuser_param)
+
+        authcfg_param = QgsProcessingParameterAuthConfig(
+            self.AUTHCFG,
+            self.tr("Authentification"),
```

Comment:
I think the term used in QGIS is `Authentication`. I leave it to you whether you want to replace all occurrences of `Authentification`.